external-crates/move/crates/move-compiler/tests/move_check/expansion/unpack_assign_block_expr.exp

```diff
@@ -1,9 +1,41 @@
+error[E04016]: too few arguments
+  ┌─ tests/move_check/expansion/unpack_assign_block_expr.move:4:9
+  │
+4 │         S { let f = 0; } = S { f: 0 };
+  │         ^^^^^^^^^^^^^ Missing argument for field 'f' in '0x42::M::S'
```

Comment:
Another situation where I would expect a resilient parser to leave the `f` here.

Reply:
I kind of think this code is straight nonsense and don't particularly care what we do lol. I defer to both of y'all